### ISSUE TITLE
question about cifar10

### ISSUE THREAD

**Issue by congxin513** [kimiyoung/ssl_bad_gan#9]
I run the code cifar_trainer.py . No changes to code, data and parameters. And the pytorch version is 0.1.12. But the result does not reach the 14.41 that you published in the paper until #400000 iterations. Could you tell me how the hyper parameters is set? Is the discriminator learning rate 6e-4 or 3e-4 by default?
